### PR TITLE
updated branch

### PR DESCRIPTION
**Pull request by 2moonyo**

### CODE REVIEW

**Comment by 2moonyo** [comment]
Changes

**Review by 2moonyo — COMMENTED** [review]
Changes

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Make sure you have the following:
 1. Python 3.x installed.
 2. pip for managing Python packages.
 3. for the required libraries please refer to 
-    requirements.txt --- install using the command pip install -r requirements.txt
+    requirements.txt --- install using the command pip install -r utils/requirements.txt
 
 
 # Usage
```
Comment (2moonyo) [inline README.md]: utils/addition

**Review by 2moonyo — COMMENTED** [review]
changes

**Review by Fatemeh992 — APPROVED** [review]
approved

**Review by Fatemeh992 — COMMENTED** [review]
approved

**Review by 2moonyo — COMMENTED** [review]
approved

**Comment by Fatemeh992** [comment]
approved

**Pull request merged**